### PR TITLE
Add docs for monitoring connectivity

### PR DESCRIPTION
**Pull request by wyb1**
**How to categorize this PR?**

/area monitoring
/area documentation
/kind enhancement

**What this PR does / why we need it**:

Add some documentation for the difference between seed and shoot connectivity.

image

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: docs/monitoring/connectivity.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## Shoot Connectivity
+
+We measure the connectivity from the shoot to the API Server. This is done via the `blackbox exporter` which is deployed in the shoot's `kube-system` namespace. Prometheus will scrape the `blackbox exporter` and then the exporter will try to access the API Server. Metrics are exposed if the connection was successful or not. This can be seen in the dashboard `Kubernetes Control Plane Status` dashboard under the `API Server Connectivity` panel. The `shoot` line represents the connectivity from the shoot.
+
+![image](images/panel.png)
+
+## Seed Connectivity
+
+In addition to the shoot connectivity, we also measure the seed connectivty. This means trying to reach the API Server from the seed via its load balancer. The connectivity is also displayed in the above panel as the `seed` line. Both `seed` and `shoot` connectivity are shown below.
```
Comment (dguendisch) [inline docs/monitoring/connectivity.md]:
```suggestion
In addition to the shoot connectivity, we also measure the seed connectivity. This means trying to reach the API Server from the seed via its load balancer. The connectivity is also displayed in the above panel as the `seed` line. Both `seed` and `shoot` connectivity are shown below.
```

**Review by dguendisch — CHANGES_REQUESTED** [review]
only nit, thanks for adding the doc

File: docs/monitoring/connectivity.md (hunk 1 above)
Comment (wyb1) [reply docs/monitoring/connectivity.md]: Thanks! Fixed!

**Review by dguendisch — DISMISSED** [review]
/lgtm

**Review by vpnachev — DISMISSED** [review]
/lgtm

**Review by rfranzke — COMMENTED** [review]
Can you please extend the index (https://github.com/gardener/gardener/blob/master/docs/README.md#monitoring)?

**Review by rfranzke — APPROVED** [review]

**Pull request merged**